.github/workflows/build-common.yml

```diff
@@ -168,10 +168,22 @@ jobs:
       - name: Upload agent jar
         uses: actions/upload-artifact@5d5d22a31266ced268874388b861e4b58bb5c2f3 # v4.3.1
         with:
-          name: opentelemetry-javaagent
+          name: opentelemetry-javaagent.jar.zip
           path: javaagent/build/libs/opentelemetry-javaagent-*-SNAPSHOT.jar
           if-no-files-found: ignore
 
+      - shell: bash
+        name: Collect SBOM's
+        run: |
+          mkdir sboms
+          find . -path ./sboms -prune -o -name "*.spdx.json" -exec mv {} ./sboms/ \;
```

Comment:
In the downloaded artifact there is a single `spdx.json` file. If that is expected then perhaps we should just copy that single file assuming that it is always generated to the same place.

Reply:
This was a bit of a carry over from the other PR where there were many files.  I kept it to make it easier to add more SBOMs in the future.  Either way will work.  Is this important to you?

Reply:
I'd vote for simplifying if we don't know we'll need more SBOMs in the future

Reply:
Ok, done.